tests/app/notifications/test_process_notifications.py

```diff
@@ -866,35 +835,6 @@ def test_persist_letter_notification_finds_correct_postage(
         notify_db_session.session.commit()
 
 
-@pytest.mark.skip(reason='Endpoint slated for removal. Test not updated.')
```

Comment:
This is wrong. Not slated for removal. This is one of those cases I warned about, they used `LETTER_TYPE` but the test is not letter-specific.